apps/parser/internal/variables/rocket_league/ranks.go

```diff
@@ -0,0 +1,239 @@
+package rocket_league
```

Comment:
Can you split variables by files please?

You can split them by logic, like one file for `rank`, one for `rating`.